File: java/notas.java
import java.util.Locale;
import java.util.Scanner;

public class notas {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double nota1, nota2, notaFinal;
		
		System.out.print("Digite a primeira nota: ");
		nota1 = sc.nextDouble();
		System.out.print("Digite a segunda nota: ");
		nota2 = sc.nextDouble();
		
		notaFinal = nota1 + nota2;
		
		if (notaFinal < 60.00) {
			System.out.println("NOTA FINAL: " + notaFinal);
			System.out.println("REPROVADO");			
		} else {
			System.out.println("NOTA FINAL: " + notaFinal);
		}
		
		sc.close();
	}

}

File: c++/notas.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {

    double primeiraNota, segundaNota, somaNotas;

    cout << "Digite a primeira nota: ";
    cin >> primeiraNota;
    cout << "Digite a segunda nota: ";
    cin >> segundaNota;

    somaNotas = primeiraNota + segundaNota;
    cout << "NOTA FINAL: " << somaNotas << endl;

    if (somaNotas < 60.0) {
        cout << "Reprovado";
    }

    return 0;
}
